renovate.json

```
@@ -1,6 +1,46 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "rebaseWhen": "behind-base-branch",
   "tekton": {
-    "schedule": ["* 0 * * *"]
+    "schedule": ["* 0 * * *"],
+    "managerFilePatterns": ["/\\.yaml$/", "/\\.yml$/"],
+    "packageRules": [
+      {
+        "description": "Enable utils image updates for tekton tasks",
+        "schedule": ["* 5-9 * * *"],
+        "matchDepTypes": ["tekton-step-image"],
+        "matchPackageNames": [
+          "quay.io/konflux-ci/release-service-utils",
+          "quay.io/conforma/cli",
+          "quay.io/konflux-ci/appstudio-utils",
+          "quay.io/konflux-ci/charon",
+          "amazon/aws-cli"
+        ],
+        "matchFileNames": [
+          "tasks/managed/**",
+          "tasks/internal/**",
+          "tasks/collectors/**"
```

Why not just tasks? That's the only three directories:
```suggestion
          "tasks/**"
```

---

fixed ty :)